contributor-docs/adrs/adr-008-experimental-components.md

```diff
@@ -47,15 +47,15 @@ The approach that has served us well since Dec 2021 has been "drafts" (along wit
 
 1. npm package: "Experimental" components, as defined by the component lifecycle, should not be part of semantically versioned npm package for `@primer/react`.
 
-2. npm package: Each experimental component should be independently published and versioned with semver. Breaking changes should be tagged as major versions. Example: `@github/react-commentbox`
+2. npm package: Each experimental component should be independently published and versioned with semver. Breaking changes should be tagged as major versions. Example: `@github/experimental-react-commentbox`
```

Comment:
I would still suggest that we create a new top level NPM org so we don't have a flood of new packages in `@github`. This would:
1. Make it easier to find them all
2. Reduce the duplication in the names (`@gre/commentbox` [GRE = GitHub React Experimental, just a random idea] as opposed to `@github/experimental-react-commentbox`)